.node/package.json

```diff
@@ -5,15 +5,16 @@
     "not dead"
   ],
   "scripts": {
-    "watch": "postcss -w ../Assets/**/*.css -o ../Resources/all.min.css --config ./postcss.config.js",
-    "build": "postcss ../Assets/**/*.css -o ../Resources/all.min.css --config ./postcss.config.js"
+    "build": "node index.js",
+    "watch:build": "nodemon -w ../Assets/ -e css -x \"npm run build\"",
+    "watch": "npm run watch:build"
```

Comment:
🤔 Could you elaborate what's the idea of this indirection?

Reply:
You're right, this extra level of indirection isn't doing anything. Fixed by 9961480.